taccsite_cms/templates/captcha/includes/js_v2_checkbox.html

```diff
@@ -0,0 +1,11 @@
+{# https://github.com/torchbox/django-recaptcha/blob/4921a6c/captcha/templates/captcha/includes/js_v2_checkbox.html #}
+{# FAQ: Just proving we can override templates (might help fix TUP-275) #}
```

Comment:
Wes, you may delete this file.

- [TUP-275](https://jira.tacc.utexas.edu/browse/TUP-275) was fixed (not with this)
- We don't need to prove we can override the template. That we can is enough. This is no longer arcane.

Reply:
Deleted: 07e1fab